src/review/views.py

```diff
@@ -19,9 +19,11 @@
 from core import models as core_models, files, forms as core_forms
 from events import logic as event_logic
 from review import models, logic, forms, hypothesis
-from security.decorators import editor_user_required, reviewer_user_required, reviewer_user_for_assignment_required, \
-    article_edit_user_required, file_user_required, article_decision_not_made, article_author_required, \
-    editor_is_not_author, senior_editor_user_required, section_editor_draft_decisions
+from security.decorators import editor_user_required, reviewer_user_required, \
+    reviewer_user_for_assignment_required,  article_edit_user_required, \
+    file_user_required, article_decision_not_made, article_author_required, \
+    editor_is_not_author, senior_editor_user_required, \
+    section_editor_draft_decisions, article_stage_review_required
```

Comment:
I can't see where "article_stage_review_required" lives
```
$ git grep "def article_stage_"
src/security/decorators.py:def article_stage_production_required(func):
src/security/decorators.py:def article_stage_accepted_or_later_required(func):
src/security/decorators.py:def article_stage_accepted_or_later_or_staff_required(func):
```

I suspect this is a new decorator that you forgot commit :P

Reply:
https://github.com/BirkbeckCTP/janeway/pull/831/files

I can see it in the commit...

Reply:
Where?

Reply:
Ah, GitHub decided to no show me the diff by default